lib/ecto/adapters/sql.ex

```diff
@@ -1071,12 +1071,6 @@ defmodule Ecto.Adapters.SQL do
     result = with {:ok, _query, res} <- result, do: {:ok, res}
     stacktrace = Keyword.get(opts, :stacktrace)
 
-    params =
-      Enum.map(params, fn
-        %Ecto.Query.Tagged{value: value} -> value
-        value -> value
-      end)
```

Comment:
I'm pretty sure this isn't needed anymore, otherwise the callbacks in the connection modules would also need to handle it?